src/main/java/org/broadinstitute/hellbender/tools/dataflow/WC.java

```diff
@@ -12,11 +12,11 @@
 import org.broadinstitute.hellbender.engine.dataflow.DataflowCommandLineProgram;
 
 @CommandLineProgramProperties(
-        usage = "Count the usages of every word in a text",
-        usageShort = "Count Words",
+        usage = "This is a test program for Dataflow via the command-line to count the usages of every word in a text",
+        usageShort = "Dataflow CLI test program for counting words",
         programGroup = DataFlowProgramGroup.class)
 
-public final class WordCount extends DataflowCommandLineProgram {
+public final class WC extends DataflowCommandLineProgram {
```

Comment:
I know it was suggested, but lets not change this to WC, it's not any harder to type WordCount, and this isn't a drop in replacement for linux WC command.